packages/client/hmi-client/src/components/workflow/scenario-templates/value-of-information/value-of-information-scenario.ts

```diff
@@ -0,0 +1,317 @@
+import { operation as SimulateCiemssOp } from '@/components/workflow/ops/simulate-ciemss/mod';
+import { operation as CompareDatasetOp } from '@/components/workflow/ops/compare-datasets/mod';
+import { operation as InterventionOp } from '@/components/workflow/ops/intervention-policy/mod';
+import { OperatorNodeSize } from '@/services/workflow';
+import {
+	createModelConfiguration,
+	getModelConfigurationById,
+	setParameterDistribution
+} from '@/services/model-configurations';
+import { ChartSetting, ChartSettingType } from '@/types/common';
+import { updateChartSettingsBySelectedVariables } from '@/services/chart-settings';
+import { AssetType, ParameterSemantic } from '@/types/Types';
+import { getInterventionPolicyById } from '@/services/intervention-policy';
+import { useProjects } from '@/composables/project';
+import { formatUniformDistribution } from '../scenario-template-utils';
+
+/*
+ * Value of information scenario
+
+	Configure the model to represent the extremes of uncertainty for some
+	parameters, then simulate into the near future with different intervention
+	policies and compare the outcomes.
+
+  Users can input a model, a model configuration, a set of interventions,
+  and a set of parameters. For each parameter, a min and max value will be specified,
+  resulting in the creation of new model configurations for each parameter with that
+	parameter set to a uniform.
+
+	Example:
+	2 interventions, 2 parameters
+  Model Node
+  |
+  +-- Model Config Node (Uniform distrubiton for Param 1)
+  |     |
+  |     +-- Simulate Node (Intervention 1)
+  |
+  +-- Model Config Node (Uniform distrubiton for Param 2)
+  |     |
+  |     +-- Simulate Node (Intervention 1)
+  |
+  +-- Model Config Node (Uniform distrubiton for Param 1)
+  |     |
+  |     +-- Simulate Node (Intervention 2)
+  |
+  +-- Model Config Node (Uniform distrubiton for Param 2)
+        |
+        +-- Simulate Node (Intervention 2)
+ */
+export class ValueOfInformationScenario extends BaseScenario {
+	public static templateId = 'value-of-information-scenario';
+
+	public static templateName = 'Value of information';
+
+	modelSpec: { id: string };
+
+	modelConfigSpec: { id: string };
+
+	interventionSpecs: { id: string }[];
+
+	simulateSpec: { ids: string[] };
+
+	parameters: (ParameterSemantic | null)[];
+
+	constructor() {
+		super();
+		this.workflowName = '';
+		this.modelSpec = {
+			id: ''
+		};
+		this.modelConfigSpec = {
+			id: ''
+		};
+		this.simulateSpec = {
+			ids: []
+		};
+		this.interventionSpecs = [{ id: '' }];
+		this.parameters = [null];
+	}
+
+	setModelSpec(id: string) {
+		this.modelSpec.id = id;
+		this.modelConfigSpec.id = '';
+		this.interventionSpecs = [{ id: '' }];
+		this.simulateSpec.ids = [];
+		this.parameters = [null];
+	}
+
+	setModelConfigSpec(id: string) {
+		this.modelConfigSpec.id = id;
+	}
+
+	setCalibrateSpec(ids: string[]) {
+		this.simulateSpec.ids = ids;
+	}
```

Comment:
The way you build the scenario, it has no calibrate? either typo or you used the wrong operator

Reply:
This was a type, I've renamed it to simulate